prisma/schema.prisma

```diff
@@ -815,6 +822,24 @@ model CollectionReport {
   @@id([reportId, collectionId])
 }
 
+model BountyReport {
+  bountyId Int
+  bounty   Bounty @relation(fields: [bountyId], references: [id], onDelete: Cascade)
+  reportId Int    @unique
+  report   Report @relation(fields: [reportId], references: [id], onDelete: Cascade)
+
+  @@id([reportId, bountyId])
```

Comment:
If we ever are to query reports for bounties than we would want to have a reverse lookup index on [bountyId]